eng/common/scripts/Save-Package-Properties.ps1

```diff
@@ -33,7 +33,7 @@ Verison property in that file.
 [CmdletBinding()]
 Param (
   [Parameter(Mandatory=$True)]
-  [string] $serviceDirectory,
+  [string] $serviceInput,
```

Comment:
It might be worth just keeping serviceDirectory and making it optional and adding a new parameter for the json file so we don't have this being a magic overloaded string.

Reply:
Ok.